editions/resumebuilder/tiddlers/Résumé Wizard.tid

```diff
@@ -1,7 +1,7 @@
 created: 20150719173441992
-modified: 20150719182326153
+modified: 20150720171426440
 tags: 
-title: Resume Wizard
+title: Résumé Wizard
```

Comment:
I had always thought (this is especially true for Windows and possibly Mac OS X) that file names should **not** have extended characters. Instead to have them *slug-ized* to standard ASCII using things like dashes (`-`) and underscores (`_`).

This is one of the reasons I thought the filename is different then the `title:` field in a tiddler. Often I will create tiddler files organized sanely with directories while the *title* has the shadow path names (`$:/core/modules/etc`).

In this case the file name should be `resume-wizard.tid` while the *title* should be `Résumé Wizard`. I hypothesize that this would be the most compatible option for the many different filesystems that TiddlyWiki source would find itself.